dubbo-plugin/dubbo-native/src/main/java/org/apache/dubbo/aot/generate/AotProcessor.java

```diff
@@ -55,6 +57,7 @@ public static void main(String[] args) {
                         ClassSourceScanner.INSTANCE.adaptiveClasses().values()))
                 .registerBeanType(ClassSourceScanner.INSTANCE.scopeModelInitializer())
                 .registerConfigType(ClassSourceScanner.INSTANCE.configClasses())
+                .registerConfigType(getCustomClasses())
```

Comment:
Instead of using registerConfigType, which only injects the contents of the method, you need to construct a method that injects the value of the field(MemberCategory.PUBLIC_FIELDS)

Reply:
Done.